### DISCUSSION TITLE
Paul Butler

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#100] [Announcements]
## Background:

[Twitter](https://twitter.com/paulgb)
[GitHub](https://github.com/paulgb)
[Website](https://driftingin.space)

https://jamsocket.com
https://github.com/drifting-in-space/spawner
https://aper.dev

## Intro

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin. 

Justin: Hey everyone! Our guest today is Paul Butler. I had the great pleasure of attending the Recurse Center with Paul in 2021. Paul has recently helped co-found Drifting in Space. We'll talk more about that in a bit, but before we dig in, Paul, would you tell our audience a little more about yourself?

## Questions

- [x] Whats the goal of Drifting in Space the company and why did you create it?
- [x] What are Session-lived Application Backends and how do they differ from traditional web apps?
- [x] What is spawner? How does it make Session-lived Application Backends easy?
- [x] What types of applications fit this architecture? What are some real world examples?
- [ ] Why do you think "All software is web software now"?
- [x] What is jam socket?
- [x] What is https://aper.dev?
- [x] What things have you built/seen built on it?
- [ ] What do you think the future of the web looks like?

## Tooltips

Add them in the comments below!

## Outro

Andrew: Well that’s it for this week’s episode of devtools.fm. Be sure to follow us on YouTube and wherever you consume your podcasts. Thanks for listening!

**Comment by paulgb** [comment]
Tooltips:

- [NATS](https://nats.io/)
- [Trunk](https://trunkrs.dev/)

**Comment by just-be-dev** [comment]
Tooltips:
- https://github.com/VirtusLab/git-machete
- https://github.com/Gerrnperl/outline-map
